flexget/plugins/modify/path_select.py

```diff
@@ -0,0 +1,216 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+import os
+import random
+from collections import namedtuple
+
+from flexget import plugin
+from flexget.event import event
+from flexget.config_schema import one_or_more
+
+log = logging.getLogger('path_select')
+
+SYMBOLS = {
+    'customary': ('B', 'K', 'M', 'G', 'T', 'P', 'E', 'Z', 'Y'),
+    'customary_ext': ('byte', 'kilo', 'mega', 'giga', 'tera', 'peta', 'exa', 'zetta', 'iotta'),
+    'iec': ('Bi', 'Ki', 'Mi', 'Gi', 'Ti', 'Pi', 'Ei', 'Zi', 'Yi'),
+    'iec_ext': ('byte', 'kibi', 'mebi', 'gibi', 'tebi', 'pebi', 'exbi', 'zebi', 'yobi'),
+}
+
+
+def human2bytes(s):
```

Comment:
Perhaps this should be a utility function for any plugin wishing to accept a file size in the config? We have a similar thing for intervals here
https://github.com/Flexget/Flexget/blob/develop/flexget/utils/tools.py#L320
I guess if we do that we should have a standard way to define one of these in a config schema, once again probably similar to how we do intervals:
https://github.com/Flexget/Flexget/blob/develop/flexget/config_schema.py#L167-L171